src/main/java/uk/gov/ch/repository/update/trusts/CorporateTrusteesRepository.java

```diff
@@ -7,14 +7,45 @@
 
 public interface CorporateTrusteesRepository extends
         PagingAndSortingRepository<CorporateTrusteeData, Long> {
-    @Query(value = "SELECT ctd.trustee_id, ctd.trustee_name, "
-            + "ctd.on_register_in_country_formed, ctd.register_location, ctd.registration_number, "
-            + "ctd.law_governed, ctd.legal_form, ctd.country, "
-            + "t.corporate_ind, t.trustee_type_id, "
-            + "t.appointment_date, t.ceased_date "
+    @Query(value = "SELECT t.trustee_id, "
```

Comment:
I believe this will return two rows for each trustee. Because we join on the trustee address link table.

Reply:
So when testing that doesn't seem to be the case

![image](https://github.com/companieshouse/oracle-query-api/assets/124157110/def8ad70-b36d-481a-a2fe-3a3e148f5aef)

Reply:
That's pointed at Reserve1 with the trust ID 3000000085 if you want to try it yourself 

Here is the JSON it produces _which I probably should have put in the description I'll do that now_

```json
[
    {
        "trusteeId": "3000000041",
        "trusteeName": "MY TRUSTED COMPANY",
        "registerLocation": "BERLIN",
        "registrationNumber": "567845",
        "lawGoverned": "GERMAN LAW",
        "legalForm": "MY Trustee OLE LEGAL FORM",
        "country": "GERMANY",
        "onRegisterInCountryFormed": true,
        "corporateIndicator": "Y",
        "trusteeTypeId": 5004,
        "appointmentDate": "2022-11-01 00:00:00.0",
        "ceasedDate": null,
        "serviceAddress": {
            "address_line_1": "OLD CHURCH ROAD",
            "address_line_2": null,
            "care_of": null,
            "country": "WALES",
            "locality": "CARDIFF",
            "po_box": null,
            "postal_code": "CF14 1AA",
            "premises": "27",
            "region": "SOUTH GLAMORGAN"
        },
        "registeredOfficeAddress": {
            "address_line_1": "OLD CHURCH ROAD",
            "address_line_2": null,
            "care_of": null,
            "country": "WALES",
            "locality": "CARDIFF",
            "po_box": null,
            "postal_code": "CF14 1AA",
            "premises": "23",
            "region": "SOUTH GLAMORGAN"
        }
    },
    {
        "trusteeId": "3000000042",
        "trusteeName": "MY SETTLED COMPANY",
        "registerLocation": "BONN",
        "registrationNumber": "563122",
        "lawGoverned": "GERMAN LAW",
        "legalForm": "OLE LEGAL FORM",
        "country": "GERMANY",
        "onRegisterInCountryFormed": true,
        "corporateIndicator": "Y",
        "trusteeTypeId": 5003,
        "appointmentDate": "2022-09-02 00:00:00.0",
        "ceasedDate": null,
        "serviceAddress": {
            "address_line_1": "OLD CHURCH ROAD",
            "address_line_2": null,
            "care_of": null,
            "country": "WALES",
            "locality": "CARDIFF",
            "po_box": null,
            "postal_code": "CF14 1AA",
            "premises": "27",
            "region": "SOUTH GLAMORGAN"
        },
        "registeredOfficeAddress": {
            "address_line_1": "WHITCHURCH ROAD",
            "address_line_2": null,
            "care_of": null,
            "country": "WALES",
            "locality": "CARDIFF",
            "po_box": null,
            "postal_code": "CF14 1AB",
            "premises": "§",
            "region": "SOUTH GLAMORGAN"
        }
    },
    {
        "trusteeId": "3000000043",
        "trusteeName": "MY BENEFICIARY COMPANY",
        "registerLocation": null,
        "registrationNumber": null,
        "lawGoverned": "GERMAN LAW",
        "legalForm": "OLE LEGAL FORM",
        "country": null,
        "onRegisterInCountryFormed": false,
        "corporateIndicator": "Y",
        "trusteeTypeId": 5002,
        "appointmentDate": "2022-09-02 00:00:00.0",
        "ceasedDate": null,
        "serviceAddress": {
            "address_line_1": "OLD CHURCH ROAD",
            "address_line_2": null,
            "care_of": null,
            "country": "WALES",
            "locality": "CARDIFF",
            "po_box": null,
            "postal_code": "CF14 1AA",
            "premises": "27",
            "region": "SOUTH GLAMORGAN"
        },
        "registeredOfficeAddress": {
            "address_line_1": "MAJOR ROAD",
            "address_line_2": null,
            "care_of": null,
            "country": "WALES",
            "locality": "CARDIFF",
            "po_box": null,
            "postal_code": "CF14 1CB",
            "premises": "9",
            "region": "SOUTH GLAMORGAN"
        }
    }
]
```

Reply:
Maybe that was my queries having tal.appointment_address_type_id in the select as I was testing. Happy to take Oracle's word for it :-)